README.md

```diff
@@ -36,8 +36,12 @@ The **docs** component visualizes the collected data with [GitHub Pages](https:/
    git remote add ghe https://git.company.com/scm/hubble
    git push -u ghe master
    ```
-1. Open [`docs/_config.yml`](docs/_config.yml) in your editor and set the `dataURL` that you noted earlier.
-   Commit and push the change to your Hubble repository.
+1. Open [`docs/_config.yml`](docs/_config.yml) in your editor and set the `dataURL` that you noted earlier. Commit and push the change to your Hubble repository:
+   ```sh
+   git add docs/_config.yml
+   git commit -m "Adjusting dataURL to our own instance"
+   git push
```

Comment:
Thanks for the contribution! 👍 

At this point I think we should recommend the user to push to `ghe` remote, no?